website/app/styles/doc-components/icons-list/index.scss

```diff
@@ -103,10 +109,36 @@
 }
 
 .doc-icons-list-grid-item__meta {
+  position: relative;
   width: 100%;
-  padding: 16px;
+  min-width: 0;
+  padding: 8px 16px;
 }
 
+.doc-icons-list-grid-item__permalink {
+  position: absolute;
+  top: 8px;
+  right: 8px;
```

Comment:
I would consider moving it farther into the corner to ad more space between the permalink and other content.

**Current 8px top & right:**
![image](https://github.com/hashicorp/design-system/assets/108769823/0e994997-dbf5-4297-af1b-b54f88305fe6)
<img width="346" alt="image" src="https://github.com/hashicorp/design-system/assets/108769823/2cfc6545-2f5a-41ad-adf4-3e51e02b6b19">

**Changed to 4px top & right:**
<img width="1046" alt="image" src="https://github.com/hashicorp/design-system/assets/108769823/8bfb0801-6bb1-416c-9f98-788808dbce84">
<img width="347" alt="image" src="https://github.com/hashicorp/design-system/assets/108769823/0a35ac19-e41f-4f61-b523-a59de0036a4a">

**Changed to 0 top & right:**
<img width="1051" alt="image" src="https://github.com/hashicorp/design-system/assets/108769823/84a701e0-0636-4315-b0f8-42e99e471bb2">
<img width="349" alt="image" src="https://github.com/hashicorp/design-system/assets/108769823/56a3b8e3-b0cf-405d-b9d8-f594cfcea6a7">

Reply:
@hashicorp/hds-design suggestions? alternative ideas?

Reply:
I like @KristinLBradley's suggestion of `4px`; a happy medium!

Reply:
updated to `4px`

Reply:
Thanks! Yes I meant both the `top` and `right` positions